<launch>

    <arg name="sim" default="true" />
    <arg name="ip" default="127.0.0.1" />
    <arg name="robot" default="ur5_camera_wall" />
    <arg name="use_camera" default="true" />

    <include file="$(find sawyer_planner)/launch/launch_base.launch">
        <arg name="sim" value="$(arg sim)" />
        <arg name="ip" value="$(arg ip)" />
        <arg name="robot" value="$(arg robot)" />
        <arg name="global_camera_cloud" value="cloud_bottom_in" />
        <arg name="inhand_camera_cloud" value="cloud_inhand_filtered" />
        <arg name="base_frame" value="world" />
    </include>

<!--     <include file="$(find realsense2_camera)/launch/rs_rgbd.launch" /> -->
    <node name="trajectory_manager" pkg="sawyer_planner" type="trajectory_recorder.py" output="screen" />
    <node name="image_recorder" pkg="sawyer_planner" type="image_recorder.py" output="screen" />
    <node name="tool_link_pub" pkg="tf" type="static_transform_publisher" args="0 0 0 -0.5 0.5 -0.5 0.5 or_ee_link or_tool_link 10" />

<!--     <node name="global_camera_pub" pkg="tf" type="static_transform_publisher" args="-0.063 0.053 -0.173 0 0.70711 0 0.70711 base_link global_camera 10" /> -->

        <node name="global_camera_pub" pkg="tf" type="static_transform_publisher" args="0.173 0.053 -0.063 0 0 0 1 base_link global_camera 10" />


    <node name="camera_pub" pkg="tf" type="static_transform_publisher" args="-0.030 -0.066 0.098 0 0 0 1 or_tool_link inhand_camera 10" />
    <param name="use_camera" value="$(arg use_camera)" />

    <node name="branch_orientation" pkg="sawyer_planner" type="branch_orientation.py" />

</launch>